common/supplemental/supplementalData.xml

```diff
@@ -4787,91 +4787,91 @@ XXX Code for transations where no currency is involved
         <calendar type="islamic">
           <calendarSystem type="lunar" />
           <eras>
-              <era type="0" start="622-7-15" code="islamic" aliases="ah"/>
+              <era type="0" start="622-7-15" code="ah"/> <!-- Anno Hegirae -->
           </eras>
         </calendar>
         <calendar type="islamic-civil">
           <calendarSystem type="lunar" />
           <eras>
-              <era type="0" start="622-7-16" code="islamic-civil" aliases="islamicc ah"/>
+              <era type="0" start="622-7-16" code="ah"/> <!-- Anno Hegirae -->
           </eras>
         </calendar>
         <calendar type="islamic-rgsa">
           <calendarSystem type="lunar" />
           <eras>
-              <era type="0" start="622-7-15" code="islamic-rgsa" aliases="ah"/>
+              <era type="0" start="622-7-15" code="ah"/> <!-- Anno Hegirae -->
           </eras>
         </calendar>
         <calendar type="islamic-tbla">
           <calendarSystem type="lunar" />
           <eras>
-              <era type="0" start="622-7-15" code="islamic-tbla" aliases="ah"/>
+              <era type="0" start="622-7-15" code="ah"/> <!-- Anno Hegirae -->
           </eras>
         </calendar>
         <calendar type="islamic-umalqura">
           <calendarSystem type="lunar" />
           <eras>
-              <era type="0" start="622-7-15" code="islamic-umalqura" aliases="ah"/>
+              <era type="0" start="622-7-15" code="ah"/> <!-- Anno Hegirae -->
           </eras>
         </calendar>
         <calendar type="chinese">
           <calendarSystem type="lunisolar"/>
           <eras>
-              <era type="0" start="-2636-01-01" code="chinese"/> <!-- 'sequential' year -->
+              <era type="0" start="-2636-01-01"/> <!-- 'sequential' year -->
           </eras>
         </calendar>
         <calendar type="hebrew">
           <calendarSystem type="lunisolar"/>
           <eras>
-              <era type="0" start="-3760-10-7" code="hebrew" aliases="am"/>
+              <era type="0" start="-3760-10-7" code="am"/> <!-- Anno Mundi -->
           </eras>
         </calendar>
         <calendar type="buddhist">
           <calendarSystem type="solar" />
           <eras>
-              <era type="0" start="-542-01-01" code="buddhist" aliases="be"/>
+              <era type="0" start="-542-01-01" code="be"/> <!-- Buddhist Era -->
           </eras>
         </calendar>
         <calendar type="coptic">
           <calendarSystem type="other"/>
           <eras>
-              <era type="0" end="284-08-28" code="coptic-inverse"/>
-              <era type="1" start="284-08-29" code="coptic"/>
+              <era type="0" end="284-08-28" code="??? TODO ???"/>
+              <era type="1" start="284-08-29" code="am" primary="true"/> <!-- Anno Martyrum -->
```

Comment:
There's a suggestion at https://github.com/tc39/proposal-intl-era-monthcode/issues/25#issuecomment-2743063952:
> the Coptic calendar should have eras BD and AD, not BM and AM

However, it's at least partly at odds with other resources I can find, all of which use "AM" or "A.M.":
* https://suscopts.org/coptic-orthodox/fasts-and-feasts/
* https://www.timeanddate.com/calendar/coptic-calendar.html
* https://en.orthodoxwiki.org/Coptic_Calendar#The_Coptic_year
* https://www.stvnashville.org/coptic-new-year#yui_3_17_2_1_1742572291617_643

I do see _references_ to A.D. (from "anno Diocletiani"), but concrete usage seems rare. I have no opinion on which is more correct here.